src/pages/variantEntity/TabSummary.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FC } from 'react';
+import Card from '@ferlab/ui/core/view/GridCard';
+import { useTabSummaryData } from 'store/graphql/variants/tabActions';
+import { Spin } from 'antd';
+
+type OwnProps = {
+  variantId: string;
+};
+
+const TabSummary: FC<OwnProps> = ({ variantId }) => {
+  const { loading } = useTabSummaryData(variantId);
+
+  return (
+    <Spin spinning={loading}>
+      <div className={'content-container'}>
+        <Card>
+          <div className={'grid-top-container'}>
+            <div className={'grid-top-container-item'}>
+              <div style={{ display: 'flex' }}>
+                <div style={{ width: '25%' }}>Chr</div>
+                <div style={{ width: '75%' }}>missing</div>
+              </div>
```

Comment:
inline component?

Reply:
that's Adrian page. I will let it as-is for now